salt/bugs/init.sls

```diff
@@ -210,7 +210,36 @@ postfix:
       - file: /etc/postfix/virtual
       - file: /etc/postfix/reject_recipients
 
+{# We can extend this for smtps/submission later #}
+{% for (port, service) in [(25, "smtp")] %}
+/etc/consul.d/roundup-{{ service }}.json:
+  file.managed:
+    - source: salt://consul/etc/service.jinja
+    - template: jinja
+    - context:
+        name: roundup-{{ service }}
+        port: {{ port }}
+    - user: root
+    - group: root
+    - mode: "0644"
+    - require:
+      - pkg: consul-pkgs
+{% endfor %}
+
 {% for tracker, config in pillar["bugs"]["trackers"].items() %}
+/etc/consul.d/roundup-{{ tracker }}.json:
```

Comment:
just as a matter of convention, we generally put this state to create a service after the thing that exposes the service. in this case it is nginx. it's probably fine but it does keep things somewhat "linear" to read.